charts/fullstack-deployment/values.yaml

```
@@ -115,6 +116,27 @@ defaults:
           require: true
           prioritize: true
       resources: {}
+    recordStreamSidecarUploader:
+      enabled: true
+      nameOverride: "record-stream-sidecar-uploader"
+      image:
+        registry: "gcr.io"
+        repository: "hedera-registry/uploader-mirror"
+        tag: "1.3.0"
+        pullPolicy: "IfNotPresent"
+      config:
+        debug: true
+        compression: true
+        sidecar: true
+        reaper:
+          enabled: true
+          minKeep: 1
+          interval: 1
+          defaultBackoff: 1
+        signature:
+          require: true
+          prioritize: true
```

```suggestion
        signature:
          require: false
          prioritize: false
```
Based on Ivo's comment: https://swirldslabs.slack.com/archives/C06FWU24B7Y/p1720445868302879

---

I've checked the bucket and it does have a sidecar directory like `.../recordstreams/record0.0.10/sidecar` and it does not contain the signature file.

---

Fixed